### PR TITLE
refactor: app-menu/content.ts example for easier local dev

### PR DESCRIPTION
**Pull request by zxt-tzx**
satcking this PR as described https://github.com/InboxSDK/InboxSDK/pull/1186#issue-2267303829

### CODE REVIEW

File: examples/app-menu/content.ts [hunk 1]
```diff
@@ -33,73 +34,82 @@ InboxSDK.load(2, 'app-menu').then(async (sdk) => {
...
+    routeID: 'custom-route-3',
+    isRouteActive: (route) => {
+      const routeID = route.getRouteID();
+      return routeID === 'custom-route-3';
+    },
+    iconUrl: {
+      lightTheme: chrome.runtime.getURL('monkey-face.jpg'),
+    },
+  })
+  const panel1 = await customItem1.addCollapsiblePanel({
```
Comment (wegry) [inline examples/app-menu/content.ts:82]: imo the most compelling part of this PR is the checks around Google Meet and/or Chat being enabled. Before accepting this PR, I'd prefer to leave the whitespace and assignment changes out.

File: examples/app-menu/content.ts [hunk 2]
```diff
@@ -33,73 +34,82 @@ InboxSDK.load(2, 'app-menu').then(async (sdk) => {
...
+    primaryButton: {
+      name: 'Monkey panel',
+      onClick: () => alert('clicked custom panel 2'),
+      iconUrl: { lightTheme: chrome.runtime.getURL('monkey.png') },
+    },
+  }));
+  if (!panel2) {
+    // make sure that your app menu is enabled by going to Gmail's `Settings > Chat and Meet` and enabling
+    // at least either Chat or Meet.
+    throw new Error('panel1 is undefined');
```
Comment (wegry) [inline examples/app-menu/content.ts:111]:
```suggestion
    throw new Error('panel2 is undefined');
```

**Review by wegry — CHANGES_REQUESTED** [review]
Left a few blocking comments here around formatting. In addition, we should add examples/*/content.ts to our prettier formatted files.

```diff
diff --git a/.prettierignore b/.prettierignore
index 37927a7e..0dc57312 100644
--- a/.prettierignore
+++ b/.prettierignore
@@ -2,6 +2,7 @@
 examples/*/*
 !**/examples/*/manifest.json
 !**/examples/*/content.js
+!**/examples/*/content.ts
 /test/data/**/*.html
 /__tests__/**/*.html
 *.snap
```

**Comment by zxt-tzx** [comment]
> Left a few blocking comments here around formatting. In addition, we should add examples/*/content.ts to our prettier formatted files.
> 
> ```diff
> diff --git a/.prettierignore b/.prettierignore
> index 37927a7e..0dc57312 100644
> --- a/.prettierignore
> +++ b/.prettierignore
> @@ -2,6 +2,7 @@
>  examples/*/*
>  !**/examples/*/manifest.json
>  !**/examples/*/content.js
> +!**/examples/*/content.ts
>  /test/data/**/*.html
>  /__tests__/**/*.html
>  *.snap
> ```

am not quite sure if my change to the `.prettierignore` file is what you meant — pls feel free to modify the PR directly!

**Review by wegry — APPROVED** [review]
Thanks for the changes.

**Pull request merged**